lib/benchee/benchmark/scenario_context.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Benchee.Benchmark.ScenarioContext do
+  defstruct [
```

Comment:
Looks like we're using the whole config when passed to the printer for a function here: https://github.com/PragTob/benchee/pull/96/files#diff-402aedd43cbd599699f2ab3157ffdf8aR34 and here: https://github.com/PragTob/benchee/pull/96/files#diff-402aedd43cbd599699f2ab3157ffdf8aR35

We also use `parallel` here: https://github.com/PragTob/benchee/pull/96/files#diff-402aedd43cbd599699f2ab3157ffdf8aR36

I think for now, leaving `config` as a whole in the `ScenarioContext` is a good first step, especially since I'm not 100% sure how all this will change with the introduction of measuring memory usage...

Ok, back to playing around with removing those duplicate values!